src/components/ComposedModal/__snapshots__/ComposedModal.story.storyshot

```diff
@@ -75,7 +76,12 @@ exports[`Storybook Snapshot tests and console checks Storyshots Watson IoT/Compo
           Lots of really wide content here...
         </div>
         <div
-          className="bx--modal-footer ComposedModal__StyledModalFooter-sc-12pj5f-2 ctEQpf"
+          className="bx--modal-footer iot--composed-modal--modal-footer"
+          inputref={
```

Comment:
[Carbon is adding this.](https://github.com/carbon-design-system/carbon/blob/master/packages/react/src/components/ComposedModal/ComposedModal.js#L235)

What I think was happening before was ModalFooter was being wrapped in StyledModalFooter, so when the code linked above was reached, the class was StyledModalFooter making "inputref: button" not clone. With the removal of StyledModalFooter, it now clones. It should also clone "closeModal" but doesn't show up on the DOM because it's a function.

It would be great if someone else could look and confirm that's what was happening as I'm not super experienced with React.

Reply:
Oh thats odd. Makes sense though

Reply:
Yeah, this is interesting - I agree that appears to be what was happening. I honestly wasn't aware that Carbon used typechecking to this degree, yet another reason to migrate away from styled-components